experimental/packages/otlp-exporter-base/src/configuration/shared-env-configuration.ts

```diff
@@ -46,7 +47,7 @@ function getTimeoutFromEnv(signalIdentifier: string) {
 function parseAndValidateCompressionFromEnv(
   compressionEnvVar: string
 ): 'none' | 'gzip' | undefined {
-  const compression = process.env[compressionEnvVar]?.trim();
+  const compression = getStringFromEnv(compressionEnvVar)?.trim();
   if (compression === '') {
     return undefined;
   }
```

Comment:
This can be simplified. `getStringFromEnv` will handle trimming and returning `undefined` if the envvar is an empty string.

```suggestion
  const compression = getStringFromEnv(compressionEnvVar);
```

Reply:
resolved in [cdebac9](https://github.com/open-telemetry/opentelemetry-js/pull/5583/commits/cdebac918c71ce96b1c115064081a5f8dec9ef42)